.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: package-plugin-workflow
+
+on: [push]
+
+jobs:
+  package-plugin-job:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        unreal: [ue-4-27, ue-5-0]
+        include:
+        - unreal: ue-4-27
+          UE_IMAGE: "ghcr.io/epicgames/unreal-engine:dev-slim-4.27"
+        - unreal: ue-5-0
+          UE_IMAGE: "ghcr.io/epicgames/unreal-engine:dev-slim-5.0.2"
```

to simplify adding & updating in the future
```suggestion
        unreal: ['4.27', '5.0.2']
```